app/controllers/api/v1/protected/base_resource_controller.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module Protected
+      # This is a JSONAPI-Resources ready controller that tests authentication using JWTSessions and tests
+      # authorization using Pundit policies.
+      class BaseResourceController < ApplicationController
+        include JSONAPI::ActsAsResourceController
+        include Pundit
+
+        # :authorize_access_request! is part of JWTSessions and is all about ensuring valid `authenticated` requests
```

Comment:
Hmm, this could be a bit misleading for the Refresh Token Endpoint, I assume that controller would not use this.